### PR TITLE
Removing latest filter Collection Version

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
closes #5227

### CODE REVIEW

File: CHANGES/5227.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Removing latest filter Collection Version.
```
Comment (bmbouter) [inline CHANGES/5227.misc]: I think this needs to be named `5227.removal` so it's listed in the breaking changes section of the release notes.

Comment (bmbouter) [reply CHANGES/5227.misc]: Also maybe put latest in single quotes, double quotes, or backticks.

File: CHANGES/5278.misc [hunk 2]
```diff
@@ -0,0 +1 @@
+Adding is_highest filter for Collection Version.
```
Comment (bmbouter) [inline CHANGES/5278.misc]: Since this is an addition that is user facing I think it should be `5278.feature`.

Comment (bmbouter) [reply CHANGES/5278.misc]: Also similar w/ the single quotes, backticks, etc.

File: pulp_ansible/app/viewsets.py [hunk 3]
```diff
@@ -105,39 +103,9 @@ def filter_by_q(self, queryset, name, value):
...
-                version_list.sort(reverse=True)
-                latest_pk = version_list[0][1]
-                latest_pks.append(latest_pk)
-
-        return queryset.filter(pk__in=latest_pks)
-
     class Meta:
         model = CollectionVersion
-        fields = ["namespace", "name", "version"]
+        fields = ["namespace", "name", "version", "is_highest"]
```
Comment (bmbouter) [inline pulp_ansible/app/viewsets.py]: Should 'q' be in here too? What happens to CollectionVersion filtering if `fields` isn't specified at all?

Comment (fabricio-aguiar) [reply pulp_ansible/app/viewsets.py]: I'm new to Django, and `q` wasn't there before, I don't know why!

File: CHANGES/5227.removal [hunk 4]
```diff
@@ -0,0 +1 @@
+Removing `latest` filter Collection Version.
```
Comment (bmbouter) [inline CHANGES/5227.removal]: :+1:

**Pull request merged**